smack-extensions/src/main/java/org/jivesoftware/smackx/muc/MultiUserChat.java

```diff
@@ -683,6 +685,7 @@ public void join(Resourcepart nickname, String password) throws XMPPErrorExcepti
      * decide to create a new room or not.
      *
      * @param mucEnterConfiguration the configuration used to enter the MUC.
+     * @return the join presence as reflected by the MUC.
```

Comment:
XEP-0045 uses the term "self-presence", hence I think this should also read self-presence (instead of just "presence).